index.html

```diff
@@ -316,7 +316,88 @@ <h3 class="about_title">
 				</ul>
 			</div>
 		</section>
+		<section class="contact_section">
+			<div class="geometry_image"></div>
+			<h3 class="about_title">
+				Get started
+			</h3>
+			<p class="about_paragraph">
+				Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
+				dolore magna.
+			</p>
+			<form action="submit">
+				<div class="input_container">
+					<label for="name" class="input_label"> Your name</label>
+					<input type="text" class="input_style" required maxlength="30" id="name">
+				</div>
+				<div class="input_container">
+					<label for="email" class="input_label"> Your email</label>
+					<input type="email" class="input_style" required id="email">
+				</div>
+				<div class="message_container">
+					<label for="msg" class="input_label">Message</label>
+					<textarea id="msg" name="user_message" class="message_area" required maxlength="500"></textarea>
+				</div>
+				<button class="collaboration_form_button">
+					Start collaboration
+				</button>
+			</form>
```

Comment:
- [ ] The requirements say that the form uses the action and method attribute to integrate with  [formspree service](https://formspree.io/html).
- action value should be similar to https://formspree.io/f/{form_id}
- method value equals POST
Check this example below 👇  

```html
        <form action="https://formspree.io/f/{form_id}" method="POST" class="contactForm">
```

- [ ] Another requirement says that you should stick to the Figma design. Kindly I would recommend you to implement this effect when the input is focused. check the difference below 👇 

Your design | The template
------------|--------------
![image](https://user-images.githubusercontent.com/65068771/148984886-3f3d4388-d469-4101-8305-05c1f47a18ef.png) | ![image](https://user-images.githubusercontent.com/65068771/148984724-563f0182-21f9-4bd4-9d3b-5d4c95ce48dc.png)